### PR TITLE
[RLlib] Give more time to impala tests

### PR DESCRIPTION
**Pull request by ArturNiederfahrenhorst**
Signed-off-by: Artur Niederfahrenhorst <artur@anyscale.com>

## Why are these changes needed?

IMPALA release tests have been flakey. They are capable of learning >> 200 mean reward on breakout, but struggle to do so within 1800s. This PR moves that bar to 2400s.
The following screenshot depicts two representative runs of how the mean reward behaves over time (1 on horizontal scale is 1h).
Note that TF has much more variance.
Screenshot 2023-01-24 at 15 22 54
I could not find out why TF has much more variance but noted that the major difference in metrics is with the value loss.
Screenshot 2023-01-24 at 15 23 28

I also tuned LR and num workers to find out of TF being faster is simply an off-policy thing. It's not.

Related TB https://tensorboard.dev/experiment/5vi5EGOdSJWdfwoc1A2qNw/#scalars&tagFilter=cur_lr&_smoothingWeight=0

### CODE REVIEW

File: release/rllib_tests/learning_tests/yaml_files/impala/impala-breakoutnoframeskip-v5.yaml [hunk 1]
```diff
@@ -18,8 +18,5 @@ impala-breakoutnoframeskip-v5:
         num_workers: 10
         num_envs_per_worker: 5
         clip_rewards: True
-        lr_schedule: [
-            [0, 0.0005],
-            [20000000, 0.000000000001],
-        ]
+        lr: 0.0005
```
Comment (ArturNiederfahrenhorst) [inline release/rllib_tests/learning_tests/yaml_files/impala/impala-breakoutnoframeskip-v5.yaml:21]: I noticed that 20000000, twntymillion in words, is never reached so to not confuse this for 2M, we should just leaved it out.

Comment (gjoliver) [reply release/rllib_tests/learning_tests/yaml_files/impala/impala-breakoutnoframeskip-v5.yaml:21]: this sounds good.

**Comment by gjoliver** [comment]
can you not leave the y scale out of the screen shot?
does it really take 4 hrs to reach 200? if we are judging based on max(mean_episode_reward), why would the variance matter?

**Comment by ArturNiederfahrenhorst** [comment]
@gjoliver The veriance does indeed not manifest in the max(mean_episode_reward).
I still found it surprising that it does that. Offpolicyness is the same.

**Comment by gjoliver** [comment]
ok, but I still have the question of why then do we need 4 hrs for this test?

**Comment by ArturNiederfahrenhorst** [comment]
@gjoliver Sorry, my mistake. I dug out the original tensorboard that you see in the picture: https://tensorboard.dev/experiment/5vi5EGOdSJWdfwoc1A2qNw/#scalars&tagFilter=cur_lr&_smoothingWeight=0
I might have swapped two screenshots when uploading.

**Comment by ArturNiederfahrenhorst** [comment]
So the run in the picture is a actually 40 minutes long.

**Comment by ArturNiederfahrenhorst** [comment]
In the file changes, I have only moved this from 30 to 40 minutes.

**Comment by gjoliver** [comment]
oh ok, man, this sounds a lot better. sorry I misread the numbers 😓

**Review by gjoliver — APPROVED** [review]

**Pull request merged**